Podcast/DiscoverTopicViewController.swift

```diff
@@ -175,16 +177,21 @@ class DiscoverTopicViewController: DiscoverComponentViewController {
         navigationController?.navigationBar.setBackgroundImage(nil, for: .default)
     }
 
-    func configureTopic() {
+    func configureTopic(isPullToRefresh: Bool = false) {
```

Comment:
Better name: canPullToRefresh. Also any reason we need the word topic in this function name? What about just "setup" or "configure".